external/os-guides/pentesting/kali.md

```diff
@@ -6,595 +6,168 @@ redirect_from:
+================================================================
+This is the recommended method.
+Easier to maintain and less demanding on resources, but you won’t have the full Kali GUI.
 
-        - quit katoolin by pressing `CRTL` + `c` keys
+If you need to install custom kernel modules (wifi drivers, …) you need to use the kernel provided by Kali instead of the kernel provided by Qubes, see [Managing VM Kernel.](/doc/managing-vm-kernel/)
 
-                What do you want to do ?> ^CShutdown requested...Goodbye...
-
-7. Clean up and update `kali` template
-
-       [user@kali ~]$ sudo apt-get dist-upgrade
-       [user@kali ~]$ sudo apt-get autoremove
-
-8. Shut down `kali` template
-
-       [user@kali ~]$ sudo shutdown -h now
-
-9. Start image
-
-        [user@dom0 ~]$ qvm-start kali
-
-10. Install tools
-
-    **Note** [Resize the template disk image][qubes-resize-disk-image] to at least 20GB if you plan on installing all packages from Kali distribution.
-
-    1. View Categories
-
-        - start katoolin
-
-                [user@kali ~]$ sudo katoolin
-
-        - select `2) View Categories`
-
-    2. Select the categories/tools you want to install
-
-        - For more information on how to use Katoolin see [How to Auto Install All Kali Linux Tools Using “Katoolin” on Debian/Ubuntu][katoolin-howto].
-
-        - **Note:** The `all` option does not work for `Information Gathering`, `Web Apps`, `Forensic Tools`, `Reverse Engineering` and `Extra`.
-
-11. Create a AppVMs based on the `kali` template
-
-    - (Optional) Attach necessary devices
-
-### Kali Linux TemplateVM from a Debian template  <a name="templatevm-from-debian3_2"/>
-
-This section will explain how to create your own [Kali] Linux TemplateVM based
-on a current stable Debian TemplateVM. The basic idea is to personalize the
-template with all the tools needed, and then spin up isolated AppVMs based on
-the template.
-
-The steps can be summarised as:
+The steps can be summarized as:
 
 1. Install Qubes stable Debian template
 2. Upgrade the template to Debian testing release
-3. Install Kali Linux through the ``kali-linux-full`` package
-4. Use the template to build AppVM so that you can maintain isolation between
-   e.g. pentesting jobs
-
-#### Get Kali Linux GPG key
+3. Add the Kali repository
+4. Update the template
 
+Get Kali Linux GPG key
+-----------------------
 **CAUTION:** Before proceeding, please carefully read [On Digital Signatures and Key Verification][qubes-verifying-signatures].
-This website cannot guarantee that any PGP key you download from the Internet is authentic.
-Always obtain a trusted key fingerprint via other channels, and always check any key you download against your trusted copy of the fingerprint.
+This website cannot guarantee that any GPG key you download from the Internet is authentic.
+Always obtain a trusted key fingerprint via other channels, and always check key you download against your trusted copy of the fingerprint.
```

Comment:
Neither of these changes is necessary.
PGP key is correct, not GPG key.
*Some* article is needed here.

Reply:
" Some article is needed here." where exactly ? 
The other points should be OK now

and thanks ! : )

Reply:
"check key you download" is ugly

Reply:
I have some difficulties with the English langage.. 
Reworded this to make it simpler